recipe/meta.yaml

```diff
@@ -1,35 +1,41 @@
-{% set version = "8.0.2" %}
-{% set sha256 = "08cb75c6e0a96836147cbfdc55580ae04d13e05d26ffbc377b4e1c68baa28b1f" %}
+{% set version = "8.0.3" %}
 
 package:
   name: ipywidgets
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/i/ipywidgets/ipywidgets-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 2ec50df8538a1d4ddd5d454830d010922ad1015e81ac23efb27c0908bbc1eece
 
 build:
   noarch: python
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - python >=3.7
     - pip
   run:
-    - python >=3.7
-    - ipython >=4.0.0
     - ipykernel >=4.5.1
-    - traitlets >=4.3.1
+    - ipython >=6.1.0
+    - jupyterlab_widgets >=3.0.4,<4.0.0
```

Comment:
i observed some icky behavior with the latest ipywidgets and an even slightly older labextension :crying_cat_face:

Reply:
indeed, this fixed the ick on https://github.com/jupyterlite/jupyterlite/pull/892